docs/snippets/all/extra_values.py

```diff
@@ -12,5 +13,5 @@
     ),
 )
 
-# Log an extra rect to set the view bounds
-rr.log("bounds", rr.Boxes2D(sizes=[3, 3]))
+# Set view bounds:
+rr.send_blueprint(rrb.Spatial2DView(visual_bounds=rrb.VisualBounds(min=[-1.5, -1.5], max=[1.5, 1.5])))
```

Comment:
why use `send_blueprint` and not setting it on `init(..., default_blueprint=...`?
(applies to all snippets that do this)

Reply:
I followed what @jleibs did in `docs/snippets/all/spatial3dview.py` in https://github.com/rerun-io/rerun/pull/6120

Reply:
<img width="765" alt="Screenshot 2024-04-29 at 17 15 07" src="https://github.com/rerun-io/rerun/assets/1148717/35de4da4-83e4-48a2-9b53-a0ed7fddbc9f">